scripts/jest/base.config.js

```diff
@@ -9,4 +9,12 @@ module.exports = {
     preset: 'ts-jest',
     testMatch: ['<rootDir>/**/__tests__/**/*.spec.(js|ts)'],
     coveragePathIgnorePatterns: ['/node_modules/', '/fixtures/', '/dist/'],
+    coverageThreshold: {
+        global: {
+            branches: 80,
+            functions: 90,
+            lines: 90,
+        },
+    },
+    testTimeout: 60000,
```

Comment:
I assume this needed to be duplicated from `scripts/jest/root.config.js` because of running Jest in each package?

Reply:
After discussing with you and PM last week during plan, removed the jest tests as part of the nx targets.